sdg/components.py

```diff
@@ -404,7 +429,7 @@ def set_precomputed_skills_data_ratio(sampling_size: float, skills_recipe: str):
                         output_dir=sdg_path,
                         taxonomy=taxonomy_path,
                         taxonomy_base=taxonomy_base,
-                        model_name=model,
+                        model_name="mixtral",
```

Comment:
Should this be hard-coded?

Reply:
Yes, because the embedded tokenizer is Mixtral.

Reply:
Yes, because the embedded tokenizer is Mixtral and that is the only one that RHEL AI supports right now from my understanding. If we add support for more in the future, we'll need to make this a parameter and have RHEL AI distribute tokenizers in a better way (not included in an ~80GB container image). The latter is tracked in:
https://issues.redhat.com/browse/RHELAIRFE-10